package_manifest.yaml

```diff
@@ -1111,3 +1111,67 @@ pulpcore_packages:
     python-whitenoise: {}
     python-yarl: {}
     python-zipp: {}
+
+foreman_release_packages:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    repoclosure_config: repoclosure/yum.conf
+    repoclosure_target_repos:
+      el8:
+        - el8-foreman-client-nightly
+      rhel7:
+        - el7-foreman-client-nightly
+      rhel6:
+        - el6-foreman-client-nightly
+      rhel5:
+        - el5-foreman-client-nightly
+      fedora29:
+        - f29-foreman-client-nightly
+      el7:
+        - el7-katello-pulpcore-nightly
```

Comment:
Does the foreman-release RPM go into pulpcore repositories?

Reply:
No, it's a mistake